cddl/intrep-actx.cddl

```diff
@@ -0,0 +1,8 @@
+ir-appraisal-context = {
+  ae : [ + ae ]
+  rv : [ rv ]
+  ev : [ ev ]
+  evs : [ evs ]
+  ACS: ACS
+  extra: any
```

Comment:
Extra in actx is meant to capture implementation-specific details. I could leave it unsaid, but I don't want to make appraisal seem like it can only consider the spec's types.

Reply:
I don't think actx should contain ar or ARS since that's a phase after the ACS is fully constructed. actx should just be for processing determined-good inputs to a final ACS.